src/bespokelabs/curator/request_processor/openai_online_request_processor.py

```diff
@@ -81,6 +81,8 @@ def __init__(
         frequency_penalty: Optional[float] = None,
         max_requests_per_minute: Optional[int] = None,
         max_tokens_per_minute: Optional[int] = None,
+        require_all_responses: bool = False,
+        max_retries: Optional[int] = None,
```

Comment:
Can you document this? What happens if I set this to 0?

Reply:
with 0 it tries only once